soccer_ipm/launch/soccer_ipm.launch

```diff
@@ -0,0 +1,14 @@
+<launch>
```

Comment:
I think everything in this launch file is redundant.
* tf_prefix has been deprecated and isn't used in ROS 2
* ``sim`` seems to be just a shortcut for ``use_sim_time``
* the taskset logic - I haven't seen this before but I you [mentioned using it in ROS 1](https://robocup.informatik.uni-hamburg.de/en/2022/07/experiences-with-ros-2-on-our-robots/)